public/data/lmac-1.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "Lucy Macartney",
+  "type": "personal",
+  "bio": "Software Developer and Technical Writer",
+  "avatar": "https://github.com/lmac-1.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/lmac-1",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/codingwithlucy",
+      "icon": "twitter"
+    },
+    {
+      "name": "Subscribe to my YouTube channel",
+      "url": "https://www.youtube.com/channel/UCcFG2cQfZRK178muVVkpnEw",
+      "icon": "youtube"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/lucymacartney",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow my blog on Hashnode",
+      "url": "https://hashnode.com/@codingwithlucy",
+      "icon": "hashnode"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Posted my first video on YouTube",
+      "date": "May 2021",
+      "icon": "youtube",
+      "color": "black",
+      "description": "My final project for CS50",
+      "url": "https://www.youtube.com/watch?v=q4hK0VSJWqg"
+    },
+    {
+      "title": "First job as a software developer",
+      "date": "March 2022",
+      "icon": "dollar",
+      "color": "black",
+      "description": "Building a web application with React, NextJS, Prisma and MongoDB"
+    },
+    {
+      "title": "Started working as a technical writer",
+      "date": "April 2022",
+      "icon": "book",
+      "color": "black",
+      "description": ""
```

I think you should remove this field if you're not going to use it, IMO.
```suggestion
      "color": "black"
```

---

On it! Will make the changes today, thanks!

---

I've removed the unnecessary line. Let me know if anything else is incorrect. Thanks!